.github/workflows/cmake.yml

```diff
@@ -175,6 +175,12 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Information about platform
+      run: |
+        sycl-ls
+        lsb_release -a
+        lspci | grep VGA
+
```

Comment:
you may consider adding `clinfo` here (but please, check if all our platforms have it installed)

Reply:
I add the whole script

Reply:
and what about `clinfo`? it provides different infos than what it's in that PMDK script

Reply:
okey